### PR TITLE
Remove award engagement metabox and move its content in awarded engagement submit metabox

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
## Description
Remove award engagement metabox and move its content in awarded engagement submit metabox
see https://github.com/gocodebox/lifterlms/pull/1771#discussion_r746055697

## How has this been tested?
manually

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

File: assets/scss/admin/metaboxes/_metabox-award-engagement.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+			-moz-osx-font-smoothing: grayscale;
+			content: "\f110";
+			position: relative;
+			top: -1px;
+			color: #8c8f94;
+			body:not(.admin-color-fresh) & {
+				color: currentColor; // Used when selecting a different admin color scheme from the default one.
+			}
+		}
+	}
```
Comment (thomasplevy) [inline assets/scss/admin/metaboxes/_metabox-award-engagement.scss:22]:
```suggestion
	}
	&.post_author_override label {
		display: inline-block;
		width: auto;;
	}
```

Visually moves the "(required)" element onto the same line which I think is probably how it should always be but I haven't dug into it. This makes it look good here.

File: includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php [hunk 2]
```diff
@@ -91,6 +231,33 @@ public function redirect_on_deletion( $location ) {
...
+	/**
+	 * Retrieve the current student id.
+	 *
+	 * @since [version]
+	 *
+	 * @param null|bool $creating Whether or not we're awarding an engagement.
+	 *                            If not provided, it'll be dynamically retrieved if the current screen's action is 'add'. See WordPress' `get_current_screen()`.
+	 * @return int
+	 */
+	private function current_student_id( $creating = null ) {
```
Comment (thomasplevy) [inline includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php:243]: Tests can (and should) be written for this method.

File: includes/admin/post-types/class.llms.meta.boxes.php [hunk 3]
```diff
@@ -50,7 +50,6 @@ public function __construct() {
 		new LLMS_Meta_Box_Engagement();
 
 		// Award Engagements.
-		new LLMS_Meta_Box_Award_Engagement();
```
Comment (thomasplevy) [inline includes/admin/post-types/class.llms.meta.boxes.php:53]: method changelog

**Review by thomasplevy — CHANGES_REQUESTED** [review]

File: assets/scss/admin/metaboxes/_metabox-award-engagement.scss (hunk 1 above)
Comment (eri-trabiccolo) [reply assets/scss/admin/metaboxes/_metabox-award-engagement.scss:22]: I was about but then I thought maybe not, and ended up choosing "not" :D

File: includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php (hunk 2 above)
Comment (eri-trabiccolo) [reply includes/admin/post-types/meta-boxes/class.llms.meta.box.award.engagement.submit.php:243]: done

**Review by thomasplevy — APPROVED** [review]

**Comment by thomasplevy** [comment]
@eri-trabiccolo <3

**Pull request merged**